py/kubeflow/kfctl/testing/ci/kfctl_e2e_workflow.py

```diff
@@ -336,7 +336,6 @@ def _build_step(self, name, workflow, dag_name, task_template,
     for t in workflow["spec"]["templates"]:
       if t["name"] == name:
         return t
-    workflow["spec"]["templates"].append(new_template)
```

Comment:
Why did this line get deleted?

Reply:
Seems `new_template ` is not defined anywhere.

Reply:
Thanks. It looks like this is legacy code and this is now handled by 
https://github.com/kubeflow/testing/blob/f5aaa62db495a78816ac31a9673c3a0ba0602c8f/py/kubeflow/testing/argo_build_util.py#L151

Can we delete everything after line 333 and just do

```
return argo_build_util.add_task_to_dag(....)
```

Reply:
updated